src/App.jsx

```diff
@@ -14,33 +16,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [activeSort, setActiveSort] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const SORT_BY_ALPHABET = 'Alphabetically';
+  const SORT_BY_LENGTH = 'ByLength';
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  const sort = typeOfSort => {
```

Comment:
a function should specify what it does. I don't see what exactly you are sorting
change the function name